features/coach_can_create_a_user.feature

```diff
@@ -0,0 +1,21 @@
+@javascript
+Feature: Coach can create a user (member)
```

Comment:
Should this be in this PR?

Reply:
It should not. I mentioned it in Slack. I made this story in a demo and it is part of https://www.pivotaltracker.com/story/show/161261034. I would like you to let this one slide.